main/config/settings/.defaults.yml

```diff
@@ -179,3 +179,6 @@ versionDate: '2021-04-19'
 
 # Metadata provided with AWS SDK calls
 customUserAgent: 'AwsLabs/SO0144/${self:custom.settings.versionNumber}'
+
+# Enable the Egress Store feature would allow researchers to securely egress data from lockdown Workspace
+enableEgressStore: false
```

Comment:
Can this be flipped back and forth? If not, is there going to be a separate task that ensures that this property cannot be changed. And if yes, how do we support it?